src/main/flight/pid.h

```diff
@@ -77,7 +75,7 @@ typedef struct pidf_s {
     uint8_t P;
     uint8_t I;
     uint8_t D;
-    uint16_t F;
+    uint16_t F; // Only keeping this for angle mode
```

Comment:
why?

Reply:
Angle mode still has access to an angle mode as the way angle mode is calculated is quite different from the regular pid controller.

Reply:
that make a better comment

Reply:
woah just read my previous comment.. man it sounds dumb. What i meant to say is that the angle mode code is calculated quite differently than the normal acro code is. FeedForward is still being used in the angle pid controller. Proved not to be useful for acro, but angle FeedForward is slighly different for a few reasons.

Reply:
i was kidding in terms of put something in the code that explains it well please